description/launch/view_robot.launch

```diff
@@ -0,0 +1,4 @@
+<launch>
+    <!-- Load the custom RViz configuration -->
+    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find description)/rviz/cybertruck.rviz"/>
+</launch>
```

Comment:
I believe we will have a file called `view.launch`, so when resolving merge conflicts, make sure to only keep one of them